### PR TITLE
Extra translateables and update russian translation

### PR DESCRIPTION
**Pull request by Blueberryy**

### CODE REVIEW

File: lua/autorun/vj_globals.lua [hunk 1]
```diff
@@ -56,7 +56,7 @@ if SERVER then
 	end
 end
 ---------------------------------------------------------------------------------------------------------------------------------------------
-function VJ_PICK(tbl)
+function VJ_PICK(tbl) -- Needs to be edited a bit to use whole line as phrase
```
Comment (DrVrej) [inline lua/autorun/vj_globals.lua:59]: Not sure what this means?

File: lua/autorun/vj_menu_spawninfo.lua [hunk 2]
```diff
@@ -36,31 +36,31 @@ VJ.AddNPC("Mortar Synth","npc_vj_mortarsynth",vCat)
 ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 VJ.AddEntity("Admin Health Kit","sent_vj_adminhealthkit","DrVrej",true,0,true,vCat)
 VJ.AddEntity("Player Spawnpoint","sent_vj_ply_spawnpoint","DrVrej",true,0,true,vCat)
-VJ.AddEntity("Fireplace","sent_vj_fireplace","DrVrej",false,0,true,vCat)
+VJ.AddEntity("FirePlace","sent_vj_fireplace","DrVrej",false,0,true,vCat)
```
Comment (DrVrej) [inline lua/autorun/vj_menu_spawninfo.lua:39]: Fireplace is the correct English word

File: lua/autorun/vj_menu_spawninfo.lua [hunk 3]
```diff
@@ -22,10 +22,10 @@ VJ.AddCategoryInfo("Fallout", {Icon = "vj_base/icons/fallout.png"})
 VJ.AddCategoryInfo("Killing Floor 1", {Icon = "vj_base/icons/kf1.png"})
 VJ.AddCategoryInfo("Left 4 Dead", {Icon = "vj_base/icons/l4d.png"})
 VJ.AddCategoryInfo("Mass Effect 3", {Icon = "vj_base/icons/masseffect3.png"})
-VJ.AddCategoryInfo("Military", {Icon = "vj_base/icons/mil1.png"})
+VJ.AddCategoryInfo("#vjbase.menu.tabs.military", {Icon = "vj_base/icons/mil1.png"})
 VJ.AddCategoryInfo("No More Room In Hell", {Icon = "vj_base/icons/nmrih.png"})
 VJ.AddCategoryInfo("Star Wars", {Icon = "vj_base/icons/starwars.png"})
-VJ.AddCategoryInfo("Zombies", {Icon = "vj_base/icons/zombies.png"})
+VJ.AddCategoryInfo("#vjbase.menu.tabs.zombies", {Icon = "vj_base/icons/zombies.png"})
```
Comment (DrVrej) [inline lua/autorun/vj_menu_spawninfo.lua:28]: These changes will broke many NPC packs

**Review by DrVrej — CHANGES_REQUESTED** [review]
To many issues in these changes, broke way to many plugins and to many files to review at once.

**Comment by Blueberryy** [comment]
Thank youf or the fix!

**Pull request closed without merging**